metis-debias/metis-debias-detect-service/src/main/resources/debias-response.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
What is this file used for?

Reply:
it was for development purposes, now it is removed.

Reply:
OK!